tensorboard/plugins/image/summary_v2.py

```diff
@@ -27,6 +27,41 @@
 def image(name, data, step=None, max_outputs=3, description=None):
     """Write an image summary.
 
+    See also `tf.summary.scalar`, `tf.summary.SummaryWriter`.
+
+    Logs a set of images to the current log directory for later analysis
+    in TensorBoard's 'Images' dashboard. Like `tf.summary.scalar` points,
+    each set of images is associated with a `step` and a `name`.  All the
+    image sets with the same `name` constitute a time series of image sets.
+
+    This example writes 2 random grayscale images:
+
+    ```python
+    test_summary_writer = tf.summary.create_file_writer('test/logs')
+    with test_summary_writer.as_default():
+      image1 = tf.random.normal(shape=[8, 8])
+      image2 = tf.random.normal(shape=[8, 8])
+      tf.summary.image("grayscale noise", [image1, image2], step=0)
+    ```
+
+    To avoid clipping, data should be converted to one of the following:
+    - floating point values in the range [0,1]
```

Comment:
Did we check the markdown rendering for this list?  (Not sure if there's an easy way to do that, maybe we should ask the docs folks.)  May work better to format as

```
To avoid clipping, data should be converted to one of the following:

- floating point values in the range [0,1], or
- uint8 values in the range [0,255]
```

Reply:
Updated, but I haven't verified the markdown rendering yet.  @wolffg would you happen to know if this markdown would render properly, or if there's a way for us to check?

I tried running some internal "docs to devsite" target but ran into a "Seems like you don't have permissions to run these kokoro builds".